doc/classes/@GDScript.xml

```diff
@@ -497,6 +512,7 @@
 			<argument index="0" name="instance" type="Object">
 			</argument>
 			<description>
+				Returns True/False whether [code]instance[/code] is a valid object (e.g. has not been deleted from memory).
```

Comment:
"Returns whether" is enough, true/false is implied.

Reply:
Took that from the doc above - fixed all occurrences.